Bottom: 445c61dd036ad4d053521ac7a7ead82d06670daa
Top:    657abc41409e8ea859f01c7640b369d8229bdd10
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 09:42:37 -0500

Point python tools to internal pypi mirror



---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
index fb6f7f4..8dc2a29 100644
--- a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
+++ b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
@@ -1,2 +1,2 @@
 [easy_install]
-index-url=https://pypi.python.org/simple
+index-url=http://172.16.103.72/repository/PyPi_Proxy/simple
